asreview/webapp/api.py

```diff
@@ -392,6 +394,23 @@ def api_random_prior_papers(project_id):  # noqa: F401
 def api_start(project_id):  # noqa: F401
     """Start training the model
     """
+
+    # get the CLI arguments
+    asr_kwargs = deepcopy(app.config['asr_kwargs'])
+
+    # add the machine learning model to the kwargs
+    # TODO@{Jonathan} validate model choice on server side
+    ml_model = request.form.get("machine_learning_model", None)
+    if ml_model and not ml_model == "cli":
```

Comment:
if ml_model != "cli"?

Isn't that equivalent/more concise?

Reply:
Thanks. I had a different intention with this part of the code. But I will propose an implementation later on. (It's not possible to experiment with new models now. So I wanted to a "CLI" model option in the front-end.)